pkgs/desktops/plasma-5/plasma-nm/default.nix

```diff
@@ -14,6 +14,10 @@ plasmaPackage {
       src = ./0001-mobile-broadband-provider-info-path.patch;
       inherit mobile_broadband_provider_info;
     })
+    (substituteAll {
+      src = ./0002-openvpn-binary-path.patch;
+      inherit openvpn;
+    })
```

Comment:
I'm merging it as is.

The irritation of updating the patches is a red herring; I've been maintaining these packages for years and I can count the number of patches I have updated on one hand.